taxonomy/constants.py

```diff
@@ -135,3 +136,8 @@ def get_job_posting_query_filter(jobs=None):
     81: 'Other Services (except Public Administration)',
     92: 'Public Administration',
 }
+
+MIN_VOTES_FOR_SKILLS = settings.MIN_VOTES_FOR_SKILLS \
+    if hasattr(settings, "MIN_VOTES_FOR_SKILLS") else 0
+RATIO_THRESHOLD_FOR_SKILLS = settings.RATIO_THRESHOLD_FOR_SKILLS \
+    if hasattr(settings, "RATIO_THRESHOLD_FOR_SKILLS") else 1
```

Comment:
@pkulkark @navinkarkera I don't have the full context here, but I agree with the suggestion here of raising an error if the settings is not configured. Otherwise, all tags could be inadvertently verified based on the current defaults.

Also, perhaps we could directly fetch the settings values in the management command instead of defining it here ?

Reply:
Yup you're right. I've updated it.